node/pkg/db/redis.go

```diff
@@ -23,106 +24,151 @@ type RedisConnectionInfo struct {
 
 var (
 	rdbMutex sync.Mutex
-	rdb      *redis.Conn
+	rdb      *redis.Client
 )
 
-func GetRedisConn(ctx context.Context) (*redis.Conn, error) {
-	conn, err := getRedisConn(ctx)
-	if err != nil {
-		return nil, err
-	}
-	return conn, err
-}
-
-func getRedisConn(ctx context.Context) (*redis.Conn, error) {
+func GetRedisConn(ctx context.Context) (*redis.Client, error) {
```

Comment:
just a minor suggestion, how about renaming all `Conn`s to `Client` for consistency?

Reply:
yes I think i missed that part, will be updated